### ISSUE TITLE
hotlink in ubuntu does not display dropdown list of actions under mouse cursor

### ISSUE THREAD

**Issue by haubourg** [xcaeag/Hotlink-QGis-Plugin#7]
neither tooltip list neither dropdown list are displayed in ubuntu + qgis 2.16.3. 

the first action of the list is launched on click then.

**Comment by haubourg** [comment]
qgis master 2 also affected.

**Comment by xcaeag** [comment]
Hi,

No pb in my Windows 1.16.3 configuration. Perhaps (Readme says) :

Attention : Since v0.7.7, Tips are not displayed because of conflict with default qgis map tip. To restore, change 'optionShowTips' variable in qgis2.ini.

[Hotlink]
optionShowTips=true

**Comment by haubourg** [comment]
ok, hotlink builds contextual menu list only if tootip display is set to "HTML" type. 
If set to a field, it fails and loads only first available action..

**Comment by haubourg** [comment]
Well, it's more complicated than that. Hotlink will build the menu well for string fields, either they are set from field's list or html expression.
If I set display field on a integer field, hotlink fails to load actions in contextual menu.
If I set that field as html expression, menu is built, but we don't have the field's value set in menu list 
ex : 
- inverser troncon
- ouvrir formulaire
trying to set html expression [% to_string(myfield) % ] does not help
If I set the display field to a text one I get:
- inverser troncon (mon texte)
- ouvrir formulaire (mon texte) 

and finally, using the same text field in an html expression will render
- inverser troncon
- ouvrir formulaire
